### PR TITLE
[Keyboard] Added Crazy Keyboard 68

### PR DESCRIPTION
**Pull request by chent7**
## Description

Added files for Crazy Keyboard 68 from taobao. Everything tested working except for caps lock led (see keyboard.c).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/crazy_keyboard_68/crazy_keyboard_68.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "crazy_keyboard_68.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	//matrix_init_user();
+	led_init_ports();
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/crazy_keyboard_68.c:23]:
```suggestion
	led_init_ports();
	
	matrix_init_user();
```

File: keyboards/crazy_keyboard_68/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     * |     |   |   |   |   |   |   |   |   |   |PSC|SLC|Pau|     |End |
+     * |----------------------------------------------------------------|
+     * |       |TOG|M_P|M_B|M_R|MSW|MSN|M_K|M_X|M_G|M_T|   |       |Ins |
+     * |----------------------------------------------------------------|
+     * |        |MOD|HUI|HUD|SAI|SAD|VAI|VAD|   |   |   |      |   |    |
+     * |----------------------------------------------------------------|
+     * |Reset|   |    |                       |   |   |    |   |   |    |
+     * `----------------------------------------------------------------'
+     */
+    [_FN] = LAYOUT(
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/keymaps/default/keymap.c:59]:
```suggestion
    [_FN] = LAYOUT_65_ansi(
```

File: keyboards/crazy_keyboard_68/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |PgUp|
+     * |----------------------------------------------------------------|
+     * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return |PgDn|
+     * |----------------------------------------------------------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|Del |
+     * |----------------------------------------------------------------|
+     * |Ctrl|Win |Alt |        Space          |Alt|Fn |Ctrl|Lef|Dow|Rig |
+     * `----------------------------------------------------------------'
+     */
+    [_BASE] = LAYOUT(
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/keymaps/default/keymap.c:38]:
```suggestion
    [_BASE] = LAYOUT_65_ansi(
```

File: keyboards/crazy_keyboard_68/info.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
+{
+    "keyboard_name": "crazy_keyboard_68",
+    "url": "",
+    "maintainer": "chent7",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 68,
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/info.json:9]:
```suggestion
        "LAYOUT_65_ansi": {
```

File: keyboards/crazy_keyboard_68/crazy_keyboard_68.h [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/crazy_keyboard_68.h:44]:
```suggestion
#define LAYOUT_65_ansi( \
```

File: keyboards/crazy_keyboard_68/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = 65_ansi
```

File: keyboards/crazy_keyboard_68/rules.mk [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
```
Comment (zvecr) [inline keyboards/crazy_keyboard_68/rules.mk:20]:
```suggestion
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/crazy_keyboard_68/config.h [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED // by default from pcb
+#define PRODUCT_ID   0x6060
```
Comment (noroadsleft) [inline keyboards/crazy_keyboard_68/config.h:24]:
```suggestion
#define PRODUCT_ID   0x13DE
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Just this, please.

File: keyboards/crazy_keyboard_68/config.h (hunk 8 above)
Comment (chent7) [reply keyboards/crazy_keyboard_68/config.h]: done :+1:

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**